README.md

```diff
@@ -441,6 +449,32 @@ Getting path to an executable based on a command name:
 console.log(await $.which("deno")); // outputs the path to deno executable
 ```
 
+Check if a command exists:
+
+```ts
+console.log(await $.commandExists("deno"));
+console.log($.commandExistsSync("deno"));
+```
+
+Check if a command does not exist:
+
+```ts
+console.log(await $.commandMissing("deno"));
+console.log($.commandMissingSync("deno"));
+```
+
+Check if an environment variable is set (and is not blank):
+
+```ts
+console.log($.envExists("HOME"));
+```
+
+Check if an environment variable is not set (or is blank):
+
+```ts
+console.log($.envMissing("HOME"));
```

Comment:
Again, don't want to add functions for the inverse of boolean results because of the API noise it adds.

Reply:
Sounds good, this API has been removed from the PR :+1: